docs/guides/3_generating_proofs.md

```diff
@@ -0,0 +1,55 @@
+# Generating Proofs For Aligned
+
+## Table of contents
+
+- [SP1](#sp1)
+    - [Dependencies](#dependencies)
+    - [How to generate a proof](#how-to-generate-a-proof)
+    - [How to get the proof verified by Aligned](#how-to-get-the-proof-verified-by-aligned)
+
+## SP1
+
+### Dependencies
+
+This guide assumes that:
+
+- sp1 prover installed (instructions [here](https://succinctlabs.github.io/sp1/getting-started/install.html))
+- sp1 project to generate the proofs
+  (instructions [here](https://succinctlabs.github.io/sp1/generating-proofs/setup.html))
+- aligned installed (instructions [here](../introduction/1_getting_started.md#quickstart))
+
+### How to generate a proof
+
+> Aligned only verifies SP1 in compressed version.
+> You can check you are using compressed by opening script/src/main.rs
+> and check that the proof is generated with `client.prove_compressed` instead of `client.prove`.
+
+First, open a terminal and navigate to the script folder in the sp1 project directory
+
+Then, run the following command to generate a proof:
+
+```bash
+cargo run --release
+```
+
+### How to get the proof verified by Aligned
+
+After generating the proof, you will have to find two different files:
+
+- proof file: usually found under `script` directory, with the name `proof.json` or similar
+- elf file: usually found under `program/elf/` directory
+
+Then, you can send the proof to the AlignedLayer network by running the following command
```

Comment:
We could change AlignedLayer for Aligned

Reply:
Changed